research/object_detection/meta_architectures/faster_rcnn_meta_arch.py

```diff
@@ -1948,9 +1950,21 @@ def _compute_second_stage_input_feature_maps(self, features_to_crop,
     Returns:
       A float32 tensor with shape [K, new_height, new_width, depth].
     """
+    num_levels = len(features_to_crop)
+    box_levels = None
+    if num_levels != 1:
+      # If there are mutiple levels to select, get the box levels
+      # unit_scale_index: num_levels-2 is chosen based on section 4.2 of
+      # https://arxiv.org/pdf/1612.03144.pdf and works best for Resnet based
+      # feature extractor.
+      box_levels = ops.fpn_feature_levels(
+          num_levels, num_levels - 2,
+          tf.sqrt(self._resize_shape[1] * self._resize_shape[2] * 1.0) / 224.0,
```

Comment:
If we add `image_shape` as an argument, we don't need `self._resize_shape` member variable